portfolio/src/main/webapp/index.html

```diff
@@ -25,84 +26,179 @@
-        </span>
-        <br />
-        <span>
-          <label for="new-event-end-time">End time:</label>
-          <input type="time" id="new-event-end-time" name="new-event-end-time" value="20:00" />
-        </span>
         <br />
-        <button type="button" class="btn btn-primary" onclick="createNewCalendarEvent()">Add this task</button>
+        <h2>Tasks added:</h2>
+        <div class="scrollable" id="new-task-list-wrapper">
+          <div id="new-task-list"></div>
+        </div>
       </div>
-    </div>
 
-    <div id="new-event-list-wrapper">
-      <h2>Events added:</h2>
-      <h3 id="event-end-time-warning">End time must be later than start time.</h3>
-      <div id="new-event-list"></div>
-    </div>
+      <div id="new-event-wrapper" class="col-md-12 col-lg-6 col-xl-3">
+        <h2>Add any pre-scheduled events here</h2>
+        <h4>We will not schedule any tasks during these time slots.</h4>
+        <form id="new-event-form">
+          <div class="form-group">
+            <label for="new-event-name">Event name:</label>
+            <input
+              type="text"
+              class="form-control"
+              aria-describedby="new-event-name"
+              id="new-event-name"
+              name="new-event-name"
+              value="New Event"
+            />
+          </div>
+
+          <div class="row">
+            <div class="col">
+              <div class="form-group">
+                <label for="new-event-start-time">Start time:</label>
+                <input
+                  type="time"
+                  class="form-control"
+                  aria-describedby="new-event-start-time"
+                  id="new-event-start-time"
+                  name="new-event-end-time"
+                  value="09:00"
+                />
+                <small id="new-event-time-help-text" class="form-text text-muted">
+                  The times you enter will be in your current timezone.
+                </small>
+              </div>
+            </div>
+            <div class="col">
+              <div class="form-group">
+                <label for="new-event-end-time">End time:</label>
+                <input
+                  type="time"
+                  class="form-control"
+                  aria-describedby="new-event-end-time"
+                  id="new-event-end-time"
+                  name="new-event-end-time"
+                  value="17:00"
+                />
+              </div>
+            </div>
+          </div>
 
-    <button type="button" class="btn btn-primary" onclick="collectAllEvents()">Start scheduling!</button>
+          <button type="button" class="btn btn-primary" onclick="createNewCalendarEventUserInput()">
+            Add this event
+          </button>
 
-    <div id="schedule-result-wrapper">
-      <h2>Your tasks are scheduled!</h2>
+          <div class="alert alert-secondary" role="alert" id="event-end-time-warning">
```

Comment:
Consider having just one alert element and using Javascript to control the error message

Reply:
Holly is dealing with this in her pr.